src/axom/mint/docs/sphinx/sections/mesh_representation.rst

```diff
@@ -0,0 +1,330 @@
+      respect to the first cell abutting to the face, denoted by,
+      :math:`\mathcal{C}_0`.
+
+   From the viewpoint of a cell, :math:`\mathcal{C}_k`, its constituent faces,
+   defined in the local node numbering of the cell, are oriented
+   according to an outward facing normal with respect to the cell,
+   :math:`\mathcal{C}_k`. For example, in :numref:`figs/faceOrientation` (a),
+   the triangle, :math:`\mathcal{C}_k`, has three faces that are oriented
+   according to an outward facing normal and defined using local node numbers
+   with respect to their cell as follows, :math:`\mathcal{F}_0=\{0,1\}`,
+   :math:`\mathcal{F}_1=\{1,2\}` and :math:`\mathcal{F}_2=\{2,0\}`
+
+   As noted earlier, a face can have at most two adjacent :ref:`Cells`,
+   denoted by :math:`\mathcal{C}_0` and :math:`\mathcal{C}_1`. By convention,
+   from the viewpoint of a face, :math:`\mathcal{F}_k`, defined using global node
+   numbers, the face is oriented according to an outward facing normal with
+   respect to the cell corresponding to :math:`\mathcal{C}_0`. As depicted in
+   :numref:`figs/faceOrientation` (b), the face denoted by :math:`\mathcal{F}_k`
+   has an outward facing normal with respect to :math:`\mathcal{C}_0`,
+   consequently it is defined as follows, :math:`\mathcal{F}_k=\{1,2\}`.
+
+.. note::
+
+    By convention, :math:`\mathcal{C}_1` is set to :math:`-1` for
+    *external boundary faces*, which are bound to a single cell.
+
+.. _Nodes:
+
+Nodes
+"""""
+
+  The :ref:`Nodes` are *zero* dimensional topological entities and hence, are the
+  lowest dimensional constituent entities of a mesh. The :ref:`Nodes` are
+  associated with the spatial coordinates of the mesh and are used in defining
+  the topology of the higher dimensional topological entities comprising the mesh,
+  such as, the :ref:`Cells`, :ref:`Faces`, etc., as discussed earlier.
+  In a sense, the :ref:`Nodes` provide the means to link the :ref:`Topology`
+  of the mesh to its constituent :ref:`Geometry` and thereby instantiate the mesh
+  in physical space.
+
+.. admonition:: Definition
+
+   A mesh node, :math:`\mathcal{n_i}`, is associated with a point,
+   :math:`p_i \in \mathbb{R}^d` and provides the means to:
+
+   #. Link the :ref:`Topology` of the mesh to its constituent :ref:`Geometry`
+
+   #. Support one or more *degrees of freedom*, evaluated at the given node
+      location.
+
+
+Notably, the nodes of a mesh are not necessarily just the *vertices* of the mesh.
+As discussed in the :ref:`PreliminaryConcepts` section, a mesh is a
+discretization of a PDE. Recall, the primary purpose of the mesh is to define
+the dicsrete locations, in both *space* and *time*, at which the
+*unknown variables* or *degrees of freedom* of the governing PDE are evaluated.
+Depending on the numerical scheme employed and the :ref:`CellTypes` used, the
+:ref:`Nodes` of a mesh may also be located at cell, face and edge centroids.
```

Comment:
Consider rephrasing this sentence to something like

Depending on the numerical scheme employed and the CellTypes used, additional mesh Nodes may be located at cell, face, and edge centroids.